FEATURE-LIFECYCLE.md

```diff
@@ -25,27 +27,31 @@ information at [Feature Lifecycle Checklist](FEATURE-LIFECYCLE-CHECKLIST.md).
   <tr>
     <td>Stability</td>
     <td>Unknown</td>
+    <td>Unknown</td>
     <td>May be buggy. Using the feature may expose bugs. Not active by default.</td>
     <td>Code is well tested. The feature is safe for production use.</td>
     <td>Code is well tested and stable. Safe for widespread deployment in production.</td>
   </tr>
   <tr>
     <td>Security</td>
     <td>Unknown</td>
+    <td>Unknown</td>
     <td>Using the feature may have obvious security vulnerabilities. Discovered vulnerabilities will not necessarily be communicated broadly.</td>
-    <td>Any discovered security vulnerabilities will be publically disclosed and patched.</td>
+    <td>Any discovered security vulnerabilities will be publiclly disclosed and patched.</td>
```

Comment:
```suggestion
    <td>Any discovered security vulnerabilities will be publicly disclosed and patched.</td>
```

Reply:
I put up another PR to get this merged in.